src/ServiceControl.Audit.Persistence/Monitoring/KnownEndpoint.cs

```diff
@@ -15,7 +15,7 @@ public class KnownEndpoint
 
         public DateTime LastSeen { get; set; }
 
-        //TODO: move to ravendb projects
+        //TODO!: move to ravendb projects
```

Comment:
Technically this should move but we have an action item to raise an issue around ID generation strategies and it would cover this so I propose to just remove the todo?

Reply:
How many places are these used? I wonder if each persistence should drop an `IGenerateIdentifiers` into IoC or something. Agreed that we don't need to fix this now.